libs/styles/styles.css

```diff
@@ -812,15 +1347,15 @@ a.static:hover {
 }
 
 /* mobile only */
-@media (max-width: 600px) {
+@media (max-width: 768px) {
   .con-button.button-justified-mobile {
     display: block;
     text-align: center;
   }
 }
 
 /* Tablet up */
-@media screen and (min-width: 600px) {
+@media screen and (min-width: 768px) {
```

Comment:
When dealing with both min and max width media queries, it's usually a good idea to use a 1px difference, otherwise there might be issues at exactly 768px viewport width. However, I see that wasn't considered before either, so maybe we don't change it right now.